### PR TITLE
CDPSDX-3344: Added update-to-dev and update-to-build commands.

### PR DESCRIPTION
**Pull request by sxxgrc**
Jira: https://jira.cloudera.com/browse/CDPSDX-3344

See below for what this change entails, but basically we have had a lot of issues updating CBD to what we need for local development. This change will hopefully allow for more flexibility and easier dev use of the application.

### CODE REVIEW

**Comment by sxxgrc** [comment]
I see that this isn't directly related to a command you can run with CBD....
I will update this to one that does but leave that change in unless a reviewer thinks it is unnecessary.

**Comment by sxxgrc** [comment]
Updated my changes to add in 2 new commands which update to latest CB on mow-dev and update to a specific build respectively.

**Comment by sxxgrc** [comment]
Whoops left in a hard coded version value fixing that now.

**Comment by sxxgrc** [comment]
Done.

**Comment by sxxgrc** [comment]
My main concern with this is that I omitted the database backup process done in the other update function because I do not know what for/why it is needed.
I have personally never used this command for updating/never backed up any databases.

File: include/deployer.bash [hunk 1]
```diff
@@ -199,6 +199,46 @@ cbd-update-snap() {
...
+
+    if [[ -z "$build" ]]; then
+        error "You did not supply a build! Update terminated"
+        _exit 1
+    fi
+
+    if [[ ${binver} != ${build} ]]; then
+        debug upgrade needed | yellow
+
+        local url=$(echo "https://public-repo-1.hortonworks.com/HDP/cloudbreak/cloudbreak-deployer_${build}_${osarch}.tgz")
```
Comment (cegganesh84) [inline include/deployer.bash:230]: nit: why echo?

`local url="https://public-repo-1.hortonworks.com/HDP/cloudbreak/cloudbreak-deployer_${build}_${osarch}.tgz"`

**Review by cegganesh84 — APPROVED** [review]
LGTM, I see that `cbd-update-snap` also does not have backup and restore procedure. Also the backup and restore could be outside of the update itself.

File: include/deployer.bash (hunk 1 above)
Comment (sxxgrc) [reply include/deployer.bash:230]: Good question, I don't see any reason for that I think I left it in by accident. Let me remove it and test again.

**Comment by sxxgrc** [comment]
Made the changes requested. I will wait a little bit (till end of day today) for anyone else to speak up and then merge this in.

**Comment by sxxgrc** [comment]
I am merging this in.

**Pull request merged**